doc/api/corepack.md

```diff
@@ -0,0 +1,94 @@
+# Corepack
+
+<!-- introduced_in=REPLACEME -->
+<!-- type=misc -->
+
```

Comment:
```suggestion

> Stability: 1 - Experimental
```

This will produce the Experimental stability banner:

![image](https://user-images.githubusercontent.com/718899/96215994-2ba6cd80-0f34-11eb-87e6-ab7f368d70f7.png)